source/clear-linux/tutorials/wp-install/wp-install.rst

```diff
@@ -3,204 +3,150 @@
-to name your website and acquire the necessary certificates. Those tasks are
-beyond the scope of this tutorial.
-
-There are also several actions you can take to harden your website from
-attacks. The security of your website and the data it contains are complex
-and ever-evolving tasks. They should be at the top of your to do list if you
-plan to expose your website to the outside world. This tutorial does not
-address security measures that you can take to harden your site but we
-strongly encourage you to look into it.
-
-Installing WordPress
---------------------
+WordPress can be installed in a variety of ways. These instructions are written for users who have installed phpMyAdmin.
 
-The folks at WordPress have created a `step-by-step guide`_ to install
-WordPress quickly and easily. We are following this procedure very closely.
+This tutorial follows these steps:
 
-We are setting up a WordPress blog which can be accessed at:
+1. Download WordPress and add components to the necessary directory. 
+2. Set up WordPress by entering information in the WordPress admin console.
+3. Run the installer to fully install WordPress on your system and create a username and password for your newly created blog. 
+
+Note that throughout this tutorial, we reference your website name as <your_website>. 
 
-http://<your_website>/blog
 
-To accomplish this setup, we must put the WordPress components in the
-:file:`/var/www/html/blog` directory.
+Install WordPress
+--------------------
+
+For this tutorial, we will create a WordPress blog that can be accessed at:
+
+http://<your_website>/blog
 
-Let us get started:
+To accomplish this setup, you must add WordPress components to the :file:`/var/www/html/blog` directory.
 
+Follow these steps:
 
-1. To move to the top-level of the website’s root directory, enter the
-   following command:
+1. To move to the top level of the website’s root directory, enter thefollowing command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       cd /var/www/html
 
 2. To download the latest version of WordPress, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo curl -O https://wordpress.org/latest.tar.gz
 
-3. To extract all the files and directories from the downloaded file,
-   enter the following command:
+3. To extract all files and directories from the downloaded file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo tar –xzvf latest.tar.gz
 
-4. To rename the top-level WordPress directory to “blog”, enter the following
-   command:
+4. To rename the top-level WordPress directory to “blog,” enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo mv wordpress blog
 
 5. To remove the downloaded tar file, enter the following command:
 
-   .. code-block:: console
+   .. code-block:: bash
 
       sudo rm latest.tar.gz
 
-Setting up WordPress
+Set up WordPress
```

Comment:
Revise to: 

Set up WordPress with Web-based GUI
-------------------------------------------

Reply:
Changed